CHANGELOG.md

```diff
@@ -8,6 +8,16 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
 
 ### Added 
 - Add the initial Xcode project.
-- Add the Swiftlint script to the build phase of the Woof target.
-- Add the Swiftformat script to the build phase of the Woof target.
+- Add the `Swiftlint` script to the build phase of the Woof target.
+- Add the `Swiftformat` script to the build phase of the Woof target.
 - Update the deployment target to 15.0+. 
+- Add `Owner` model to the model layer.
+- Add `Sitter` model to the model layer.
```

Comment:
nit:

It's a good practice to put link to the PR:

```suggestion
- Add `Sitter` model to the model layer - [#xx](link-here).
```

That helps to find corresponding PRs later easily.

Reply:
Added the links